src/main/webapp/WEB-INF/drivers/get_all_drivers_with_delete_button.jsp

```diff
@@ -0,0 +1,27 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Get all drivers with delete button</title>
+</head>
+<body>
+<h1>List of drivers:</h1>
+<table>
+    <tr>
+        <td>Id</td>
+        <td>Name</td>
+        <td>License Number</td>
+        <td>DELETE</td>
+    </tr>
+    <c:forEach items="${drivers}" var="driver">
+        <tr>
+            <td><c:out value="${driver.id}" /> </td>
+            <td><c:out value="${driver.name}" /> </td>
+            <td><c:out value="${driver.licenseNumber}" /> </td>
+            <td><a href="${pageContext.request.contextPath}
+            /drivers/delete?id=${driver.id}">delete this driver</a></td>
+        </tr>
+    </c:forEach>
+</table>
+</body>
+</html>
```

Comment:
It's not clear to me why do you need this jsp. For drivers it's enough to have 'add' and 'all' pages